cmd/operator/main.go

```diff
@@ -23,11 +24,16 @@ import (
 
 	kubearchivev1alpha1 "github.com/kubearchive/kubearchive/cmd/operator/api/v1alpha1"
 	"github.com/kubearchive/kubearchive/cmd/operator/internal/controller"
+	"github.com/kubearchive/kubearchive/pkg/observability"
 
 	sourcesv1 "knative.dev/eventing/pkg/apis/sources/v1"
 	//+kubebuilder:scaffold:imports
 )
 
+const (
+	otelServiceName = "kubearchive.operator"
+)
```

Comment:
As there is only one constant maybe you can put it like:

```suggestion
const otelServiceName = "kubearchive.operator"
```

Reply:
Fixed.